infrastructure/modules/aws/eks/outputs.tf

```diff
@@ -32,19 +37,28 @@ output "kms_key_id" {
   }
 }
 
+
+# TODO: ARNAUD.L : we must return list of iam_role_name,
+# 1 for each group in : self_managed_node_groups
+# and maybe todo later : 1 more for each group in eks_managed_node_groups
 output "worker_iam_role_name" {
   description = "EKS worker IAM role name"
-  value       = module.eks.worker_iam_role_name
+  #value       = module.eks.self_managed_node_groups["worker-c5.4xlarge-spot"].iam_role_name
+  #value =       try(values(module.eks.self_managed_node_groups).1.iam_role_name, "")
+  value = module.eks.cluster_iam_role_name
```

Comment:
Create two outputs:
* worker_iam_role_name: list of iam_role_name of all sef_managed_node_group
* cluster_iam_role_name or eks_iam_role_name: value = module.eks.cluster_iam_role_name

Reply:
ok

Reply:
Not yet corrected

Reply:
worker_iam_role_name deleted